src/test/kotlin/TestWorld.kt

```diff
@@ -0,0 +1,32 @@
+import io.hsar.wh40k.combatsimulator.model.MapPosition
+import io.hsar.wh40k.combatsimulator.model.UnitInstance
+import io.hsar.wh40k.combatsimulator.model.World
+import io.hsar.wh40k.combatsimulator.model.unit.*
+import org.junit.Test
+import org.junit.jupiter.api.TestInstance
+import kotlin.test.assertEquals
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
This isn't necessary IIRC

Reply:
That's fine, an article I googled suggested it was the 'Kotlin' way to do it. Happy to delete.

Reply:
The default test instance lifecycle is per method, and while this is a bit more optimised, it doesn't matter for the purposes of your tests.

Reply:
Updated